examples/private_ai_studio/README.md

```diff
@@ -200,8 +200,7 @@ module "aihub" {
   kind                    = "Hub"
   workspace_friendly_name = "Private AI Studio Hub"
   workspace_managed_network = {
-    isolation_mode = "AllowInternetOutbound"
-    spark_ready    = true
+    isolation_mode = "AllowOnlyApprovedOutbound"
```

Comment:
Is there a reason you updated the Private AI Studio Example?

Reply:
thanks for review @cmaclaughlin. i updated as it is private ai studio. we can revisit the other examples for secure access.